src/actors/equilibrium/tequila_actor.jl

```diff
@@ -187,7 +187,8 @@ function tequila2imas(shot::TEQUILA.Shot, dd::IMAS.dd{D}, par::FUSEparameters__A
     # MXH flux surface parametrization
     eq2d = eqt.profiles_2d[1]
     eq2d.grid.dim1 = shot.ρ
-    eq2d.grid.dim2 = vcat([0.0, 0.0, 0.0, 0.0, 0.0], fill(1.0, (shot.M,)), fill(-1.0, (shot.M,)))
+    L = length(shot.surfaces[1].c)
+    eq2d.grid.dim2 = vcat([0.0, 0.0, 0.0, 0.0, 0.0], fill(1.0, (L,)), fill(-1.0, (L,)))
```

Comment:
I've tested the current change, but it resulted in the following error

```
type Float64 has no field c
Stacktrace:
  [1] getproperty
    @ ./Base.jl:49 [inlined]
  [2] tequila2imas(shot::TEQUILA.Shot{Int64, StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, Matrix{Float64}, Matrix{Float64}, TEQUILA.Profile{FiniteElementHermite.FE_rep{Vector{Float64}, Vector{Float64}}, FiniteElementHermite.FE_rep{Vector{Float64}, Vector{Float64}}}, Nothing, Nothing, Nothing, TEQUILA.Profile{FiniteElementHermite.FE_rep{Vector{Float64}, Vector{Float64}}, FiniteElementHermite.FE_rep{Vector{Float64}, Vector{Float64}}}, Float64, Float64, Float64, FiniteElementHermite.FE_rep{StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, Vector{Float64}}, Vector{FiniteElementHermite.FE_rep{StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, Vector{Float64}}}, TEQUILA.QuadInfo{Vector{Float64}, Vector{SparseArrays.SparseVector{Float64, Int64}}, Matrix{Float64}, StepRangeLen{Float64, Base.TwicePrecision{Float64}, Base.TwicePrecision{Float64}, Int64}, Matrix{Float64}, Matrix{Float64}}, Vector{PreallocationTools.DiffCache{Vector{Float64}, Vector{Float64}}}, BandedMatrices.BandedLU{Float64, BandedMatrices.BandedMatrix{Float64, Matrix{Float64}, Base.OneTo{Int64}}}}, dd::IMASdd.dd{Float64}, par::FUSE.FUSEparameters__ActorTEQUILA{Float64}; ψbound::Float64, green_model::Symbol)
    @ FUSE ~/.julia/dev/FUSE/src/actors/equilibrium/tequila_actor.jl:190
  [3] _finalize(actor::FUSE.ActorTEQUILA{Float64, Float64})
    @ FUSE ~/.julia/dev/FUSE/src/actors/equilibrium/tequila_actor.jl:140
...
...
```

I've tried the following thing as you might intend, and it works fine.
Could you please check this? 
Thank you very much.

```suggestion
    N_MXH = length(IMAS.MXH(shot.surfaces[:, end]).c)
    eq2d.grid.dim2 = vcat([0.0, 0.0, 0.0, 0.0, 0.0], fill(1.0, (N_MXH,)), fill(-1.0, (N_MXH,)))
```